confluent-kafka-images.yaml

```diff
@@ -56,8 +56,6 @@ subpackages:
 
 update:
   enabled: true
-  version-transform:
-    - match: "-"
-      replace: "."
-  release-monitor:
-    identifier: 371736
+  github:
+    identifier: confluentinc/kafka-images
+    strip-prefix: v
```

Comment:
Looks like we need to keep release monitor here, that org has an IP allowlist

Reply:
Oh okay.